br-ext/board/stmicroelectronics/stm32mp1-tz/overlay-STM32MP157C-EV1-rpmb/etc/init.d/S30optee

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+DAEMON="tee-supplicant"
+DAEMON_PATH="/usr/sbin"
+DAEMON_ARGS="-d /dev/teepriv0"
+PIDFILE="/var/run/$DAEMON.pid"
+
+start() {
+	# tee-supplicant and the client applications need not run as
```

Comment:
`tee-supplicant` is run as root bellow so this comment needs to be modified.
Don't you have any issues with the data store when `tee-supplicant` is run as root since `/data/tee` is 0770 with owner:group `tee:tee`?

Reply:
> `tee-supplicant` is run as root bellow so this comment needs to be modified.

Right.

> Don't you have any issues with the data store when tee-supplicant is run as root since /data/tee is 0770 with owner:group tee:tee?

No issues experienced. Root has full access whatever the access right of files are, IIUC.

> Also if we could avoid duplicating the whole file it would be great.

I made it simple by implementing my own initd file.